pkg/aws/vpcendpoint/reconciler_normal.go

```diff
@@ -15,6 +15,10 @@ import (
 	"github.com/giantswarm/aws-vpc-operator/pkg/errors"
 )
 
+const (
+	VpcEndpointMode string = "aws.giantswarm.io/vpc-endpoint-mode"
```

Comment:
Please, move the annotation to [this repository](https://github.com/giantswarm/k8smetadata) instead. We use that repository so we can share labels and annotations across different repositories.